tests/testthat/test-geoknife_utils.R

```diff
@@ -58,6 +58,22 @@ test_that("result fails with failed job",{
   expect_error(result(failed.id), "processing is incomplete or has failed. See check()")
 })
 
+# test_that("percent complete works",{
```

Comment:
Well... it's commented out. 

I want to see if I can mock this and actually test it somehow, but not sure how to do it right no.